pulp_maven/tests/functional/api/test_crud_remotes.py

```diff
@@ -40,6 +40,7 @@ def test_remote_crud_workflow(maven_remote_api_client, gen_object_with_cleanup,
 
 
 @pytest.mark.parallel
+@pytest.mark.skip("Can't be properly tested without disabling client-side validation.")
```

Comment:
For these, I think converting them to unit tests is best. I got this suggestion from @mdellweg on Matrix (after I asked) and I agree with it. Can you convert these two to unit tests?